core/chains/evm/gas/models.go

```diff
@@ -75,24 +85,24 @@ func NewEstimator(lggr logger.Logger, ethClient evmclient.Client, cfg Config, ge
 	switch s {
 	case "Arbitrum":
 		newEstimator = func(l logger.Logger) EvmEstimator {
-			return NewArbitrumEstimator(lggr, geCfg, ethClient, ethClient)
+			return NewArbitrumEstimator(lggr, geCfg, ethClient)
 		}
 	case "BlockHistory":
 		newEstimator = func(l logger.Logger) EvmEstimator {
 			return NewBlockHistoryEstimator(lggr, ethClient, cfg, geCfg, bh, *ethClient.ConfiguredChainID())
```

Comment:
Noticed we had to add `ConfiguredChainID` to the interface just for this builder. I think the `ChainID` should be available upstream in the `Config` object. I don't want scope creep to hold this PR up anymore so I think it's ok the way it is. Just adding this note here so anyone else reviewing knows we considered it.